content/applications/marketing/marketing_automation/getting_started/first_campaign.rst

```diff
@@ -1,70 +1,112 @@
 
-To add a :guilabel:`Filter` to the target audience, click :guilabel:`Add Filter`, and a node field
-appears. In the node field, a custom equation can be configured for Odoo to use when filtering who
-to include (and exclude) in this specific marketing campaign.
+.. example::
+   If :guilabel:`Customers` is chosen as the :guilabel:`Target`, select :guilabel:`Email` in the
+   :guilabel:`Unicity based on` field so Odoo only processes one record for each customer email
+   address.
 
-.. image:: first_campaign/filter-node.png
-   :align: center
-   :alt: A filter node in Odoo Marketing Automation.
+Select :guilabel:`Search More...` from the :guilabel:`Unicity based on` drop-down menu to reveal all
+129 options in a pop-up window.
+
+Lastly on the campaign form is the :guilabel:`Filter` field. This is where a more specific audience
+filter can be configured to add specificity to the recipients of the marketing automation campaign.
+
+If left alone, the :guilabel:`Filter` field reads: :guilabel:`Match all records`. That means Odoo
+uses the :guilabel:`Target` and :guilabel:`Unicity based on` fields to determine who the recipients
+will be. The number of recipients is represented beneath as :guilabel:`record(s)`.
+
+Campaign filter rules
+---------------------
 
-When the first field of the node is clicked, a nested drop-down menu of options appears on the
-screen, wherein specific criteria is chosen based on needs of the campaign. The remaining fields on
-the node further define the criteria determining which records to include (or exclude) in the
-execution of the campaign.
+To add a more specific filter to a marketing automation campaign, click the :guilabel:`Add
+condition` button in the :guilabel:`Filter` field. Doing so reveals a series of other configurable
+filter rule fields.
 
-To add another node, click the :guilabel:`➕ (plus sign)` icon to the right of the filtering rule. To
-add a branch of multiple nodes at the same time, click the :guilabel:`⋯ (ellipses)` icon.
+In the rule fields, custom equations can be configured for Odoo to use when filtering who to include
+or exclude in this specific marketing campaign.
 
-For further information on filters, refer to :doc:`this documentation page
-</applications/marketing/marketing_automation/getting_started/target_audience>`.
+.. image:: first_campaign/filter-node-equation-fields.png
+   :align: center
+   :alt: How the filter rule equation fields look in Odoo Marketing Automation campaigns.
 
 .. note::
-   :guilabel:`Records` represent the number of contacts in the system that fit the specified
-   criteria for a campaign.
+   :guilabel:`Records` refer to contacts in the system that fit the specified criteria for a
+   campaign.
+
+Also, once :guilabel:`Add condition` is clicked, the ability to :guilabel:`Save as
+Favorite Filter` becomes available on the campaign form.
+
+.. image:: first_campaign/save-favorite-filter-option.png
```

Comment:
Not useful unless we include some more surrounding UI to provide a navigational queue as to its location. Would change or remove.